utp_util.go

```diff
@@ -0,0 +1,55 @@
+package manet
```

Comment:
this should be a package on its own (can be in this same repo), and keep things like:

```
utp.Addr
utp.Conn
utp.Dialer
utp.Listener
```
(i wouldn't do `utp.UTPConn` anymore, annoying sideeffect of `net` pkg bunching up everything)